uint32 motorIndex
bool powerStatus
---
